Tests/TuistGeneratorTests/Generator/InfoPlistContentProviderTests.swift

```diff
@@ -23,7 +23,8 @@ final class InfoPlistContentProviderTests: XCTestCase {
         let target = Target.test(
             destinations: [.iPhone, .iPad, .macWithiPadDesign],
             product: .app,
-            deploymentTargets: .iOS("16.0")
+            deploymentTargets: .iOS("16.0"),
+            sources: ["/Example.swift"]
```

Comment:
This change is no longer necessary as we expect the `CFBundleExecutable` to still be included with the latest changes

```suggestion
            deploymentTargets: .iOS("16.0")
```